### PR TITLE
Add new pages for Sheets and functions and operators

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- Add documentation and images for new Sheet product

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/sheets/_index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+
+## Further reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+
+[1]: https://app.datadoghq.com/sheets
+[2]: https://app.datadoghq.com/logs
+[3]: PLACEHOLDER
```
Comment (estherk15) [inline content/en/sheets/_index.md:81]:
```suggestion
[3]: /sheets/functions_operators
```

File: content/en/sheets/_index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Pivot table
+
+After you add a table of data to a spreadsheet, analyze and add context to your raw data with a Pivot table. Use pivot tables to summarize and organize large amounts of data into customized tables. It helps you analyze data to find patterns and trends, and see comparisons. For example, you can have a table of Error logs with a hundred rows, but with a pivot table you can break down that data into a summary table that counts your error logs by method or region. To create a pivot table:
+1. From an existing spreadsheet that already has a table of data click **Add Pivot Table**.
+1. In the Rows section, select the dimensions you want to analyze, for example, Status of logs.
+1. In the Calculations sections, select the dimensions you want to perform some calculations, including sum, average, count, min, and max.
```
Comment (jhgilbert) [inline content/en/sheets/_index.md:61]:
```suggestion
1. In the **Calculations** section, select the dimensions you want to use in calculations, including sum, average, count, min, and max.
```
Or "the dimensions you want to perform calculations on"?

File: content/en/sheets/functions_operators.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+title: Functions and Operators
+---
+
+## Overview
+
+Use functions and operators in your Sheets Calculated Columns to add additional analysis and context to your data.
```
Comment (jhgilbert) [inline content/en/sheets/functions_operators.md:7]:
```suggestion
Use functions and operators in your Sheets calculated columns to add additional analysis and context to your data.
```

File: content/en/sheets/functions_operators.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+| MOD(number1, number2)                                                  | Returns the result of the modulo operator, the remainder after a division operation.                                              | MOD(5, 2)                                                 |
+| POWER(number, power)                                                   | Returns a number raised to a power.                                                                                               | POWER(2, 3)                                               |
+| LEN(string)                                                            | Returns the length of a string.                                                                                                   | LEN("Hello World")                                        |
+| LOWER(string)                                                          | Return the string in lowercase.                                                                                                   | LOWER("HELLO WORLD")                                      |
```
Comment (jhgilbert) [inline content/en/sheets/functions_operators.md:48]:
```suggestion
| LOWER(string)                                                          | Returns the string in lowercase.                                                                                                   | LOWER("HELLO WORLD")                                      |
```

File: content/en/sheets/functions_operators.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+| UPPER(string)                                                          | Return the string in uppercase.                                                                                                   | UPPER("hello world")                                      |
```
Comment (jhgilbert) [inline content/en/sheets/functions_operators.md:49]:
```suggestion
| UPPER(string)                                                          | Returns the string in uppercase.                                                                                                   | UPPER("hello world")                                      |
```

File: content/en/sheets/_index.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+  text: "Functions and Operators"
+---
+
+{{< callout url="https://www.datadoghq.com/private-beta/datadog-sheets/" btn_hidden="false" header="Join the Beta!">}}
+Sheets is available in <strong>private beta</strong>. To qualify for this product beta, you should have existing use cases where you currently use spreadsheets (such as Excel or Google Sheets) with Datadog’s metrics, logs, or RUM data. If you're interested in this feature, complete the form to request access.
+{{< /callout >}}
+
+## Overview
+
+Sheets is a spreadsheet tool that connects to Datadog data, enabling you to perform complex data analysis and build reports without requiring technical expertise. It allows teams to use familiar spreadsheet functions like lookups, pivot tables, and calculations on Datadog data, so you don’t have to export and use another tool with stale data.
```
Comment (jhgilbert) [inline content/en/sheets/_index.md:15]:
```suggestion
Sheets is a spreadsheet tool that you can populate with Datadog data, enabling you to perform complex analysis and build reports without requiring technical expertise. It allows teams to use familiar spreadsheet functions like lookups, pivot tables, and calculations on Datadog data, so you don’t have to export and use another tool with stale data. 
```
"connects to" made me think it was more of a Google Sheets integration or similar, so I tried to reword to make the tool's purpose clearer. It's a tough one to write, though!

File: config/_default/menus/main.en.yaml [hunk 7]
```diff
@@ -1403,12 +1398,28 @@ menu:
...
       identifier: ddsql_editor_guide
       weight: 10
+    - name: Sheets
+      url: sheets/
+      pre: inventories
+      identifier: sheets
+      parent: platform_heading
+      weight: 40000
+    - name: Functions and Operators
+      url: sheets/functions_and_operators/
```
Comment (jhgilbert) [inline config/_default/menus/main.en.yaml:1408]:
```suggestion
      url: sheets/functions_operators/
```

**Review by jhgilbert — APPROVED** [review]
Approved with some comments, but happy to take another look anytime if that's helpful!

File: config/_default/menus/main.en.yaml [hunk 8]
```diff
@@ -1403,12 +1398,28 @@ menu:
...
+    - name: Functions and Operators
+      url: sheets/functions_operators/
+      parent: sheets
+      identifier: sheets_functions_and_operators
+      weight: 1
+    - name: Guides
+      url: sheets/guide/
+      parent: sheets
+      identifier: sheets_guide
+      weight: 2
```
Comment (estherk15) [inline config/_default/menus/main.en.yaml:1416]:
```suggestion
```

**Pull request merged**